mm2src/mm2_main/src/ordermatch_tests.rs

```diff
@@ -1828,8 +1828,8 @@ fn test_request_and_fill_orderbook() {
                 let orders = orders
                     .into_iter()
                     .map(|(uuid, order)| {
-                        if let Some(conf_settings) = order.conf_settings {
-                            conf_infos.insert(uuid, conf_settings);
+                        if let Some(ref conf_settings) = order.conf_settings {
+                            conf_infos.insert(uuid, conf_settings.clone());
```

Comment:
That is because of the [recomendation](https://github.com/KomodoPlatform/atomicDEX-API/pull/1768#discussion_r1231712172) to prefer to not use copy when possible.